internal/handshaker/service/service_test.go

```diff
@@ -19,6 +19,7 @@
 package service
 
 import (
+	"context"
```

Comment:
Can we add a unit test where the transport credentials are non-nil?

Reply:
done